scripts/update-chart.sh

```diff
@@ -0,0 +1,258 @@
+#!/usr/bin/env bash
+
+# update-chart.sh handles creating a new version of the parent chart, updating
+# its dependencies and pushing the chart into the repository.
+
+# In order to compare chart dependencies, we transform the information into a
+# JSON map of service to version. Then we can use jq, which has more powerful
+# comparison operators than bash to determine the difference.
```

Comment:
Would be nice to see an example invocation of this command here at the top of the file.